packages/block-editor/src/components/block-button/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import RichText from '../rich-text';
+
+export default class BlockButton extends Component {
```

Comment:
Should it be `ButtonElement` instead? (It's not a block)

Reply:
I went with ElementButton, so that we can group all the element components together.